addons/web/static/src/js/fields/relational_fields_mobile.js

```diff
@@ -0,0 +1,122 @@
+odoo.define('web.relational_fields_mobile', function (require) {
+"use strict";
+
+var config = require('web.config');
+var core = require('web.core');
+var relational_fields = require('web.relational_fields');
+
+var _t = core._t;
+
+if (!config.device.isMobile) {
+    return;
+}
+
+/**
+ * Override the Many2One to prevent autocomplete and open kanban view in mobile for search.
+ */
+
+relational_fields.FieldMany2One.include({
```

Comment:
Could we move this part in enterprise because it's mobile

Reply:
@adr-odoo We do not move code based on mobile, we have lots of code related to mobile in community, we only move if it is specifically related to web_mobile module, if you check kanban column implementation written in community